### PR TITLE
Cleanup code for MM_IdleGCManager

### PR DESCRIPTION
**Pull request by dmitripivkine**
- using proper types
- code formatting

### CODE REVIEW

**Comment by dmitripivkine** [comment]
fixes https://github.com/eclipse-openj9/openj9/issues/20453

File: runtime/gc_base/IdleGCManager.cpp [hunk 1]
```diff
@@ -88,12 +88,13 @@ MM_IdleGCManager::manageFreeHeap(J9VMThread* currentThread)
...
 
 extern "C" {
-void idleGCManagerVMStateHook(J9HookInterface** hook, uintptr_t eventNum, void* eventData, void* userData)
+void
+idleGCManagerVMStateHook(J9HookInterface **hook, uintptr_t eventNum, void *eventData, void *userData)
 {
-	J9VMRuntimeStateChanged* j9VMState = (J9VMRuntimeStateChanged*)eventData;
-	MM_IdleGCManager* idleMgr = (MM_IdleGCManager*)userData;
+	J9VMRuntimeStateChanged *j9VMState = (J9VMRuntimeStateChanged *)eventData;
+	MM_IdleGCManager* idleMgr = (MM_IdleGCManager *)userData;
```
Comment (amicic) [inline runtime/gc_base/IdleGCManager.cpp:95]: move * in `MM_IdleGCManager *idleMgr`

File: runtime/gc_base/IdleGCManager.cpp [hunk 2]
```diff
@@ -49,36 +49,36 @@ MM_IdleGCManager::newInstance(MM_EnvironmentBase* env)
...
 }
 
 void
-MM_IdleGCManager::manageFreeHeap(J9VMThread* currentThread)
+MM_IdleGCManager::manageFreeHeap(J9VMThread *currentThread)
 {
-	MM_EnvironmentBase* env = MM_EnvironmentBase::getEnvironment(currentThread->omrVMThread);
-	MM_GCExtensions* _extensions = MM_GCExtensions::getExtensions(env);
+	MM_EnvironmentBase *env = MM_EnvironmentBase::getEnvironment(currentThread->omrVMThread);
+	MM_GCExtensions *_extensions = MM_GCExtensions::getExtensions(env);
```
Comment (amicic) [inline runtime/gc_base/IdleGCManager.cpp:81]: _extensions is declared as a local, so no leading '_' is needed

Comment (amicic) [reply runtime/gc_base/IdleGCManager.cpp:81]: one general comment that we should probably agree on.... often use env and ext as local.
we should either use them in that short form or long form (environment, extension), at least in same  method, preferably same file, ideally everywhere

File: runtime/gc_base/IdleGCManager.cpp [hunk 3]
```diff
@@ -88,12 +88,13 @@ MM_IdleGCManager::manageFreeHeap(J9VMThread* currentThread)
...
+idleGCManagerVMStateHook(J9HookInterface **hook, uintptr_t eventNum, void *eventData, void *userData)
 {
-	J9VMRuntimeStateChanged* j9VMState = (J9VMRuntimeStateChanged*)eventData;
-	MM_IdleGCManager* idleMgr = (MM_IdleGCManager*)userData;
+	J9VMRuntimeStateChanged *j9VMState = (J9VMRuntimeStateChanged *)eventData;
+	MM_IdleGCManager* idleMgr = (MM_IdleGCManager *)userData;
 
-	/* idle gc manager currently handles only ACTIVE -> IDLE transition*/
+	/* idle gc manager currently handles only ACTIVE -> IDLE transition */
 	if (j9VMState->state == J9VM_RUNTIME_STATE_IDLE) {
```
Comment (amicic) [inline runtime/gc_base/IdleGCManager.cpp:98]: J9VM_RUNTIME_STATE_IDLE on left side

**Comment by dmitripivkine** [comment]
@amicic Thank you, fixed.

**Comment by amicic** [comment]
Jenkins compile win jdk11

**Review by amicic — APPROVED** [review]

**Pull request merged**